src/components/App.tsx

```diff
@@ -3061,6 +3094,10 @@ class App extends React.Component<AppProps, AppState> {
       );
     } else if (this.state.activeTool.type === "custom") {
       setCursor(this.canvas, CURSOR_TYPE.CROSSHAIR);
+      this.props?.onCustomElementPointerDown?.(
+        this.state.activeTool,
+        pointerDownState,
+      );
```

Comment:
`activeTool.type === "custom"` doesn't necessarily mean an element so we should come up with a different name for the API. `onCustomToolPointerDown`? Perhaps we should just do `onPointerDown`, and either calling for all tools, or just for `custom` (for now), despite the name.